Makefile.build

```diff
@@ -670,6 +670,16 @@ endif
 	$(SHOW)'COQDEP    $<'
 	$(HIDE)$(COQDEPBOOT) -boot $(DEPNATDYN) "$<" $(TOTARGET)
 
+# Below we provide those dependecies that are not discovered {coq,ocaml}dep programs.
+
+plugins/micromega/micromega.cmi \
+plugins/micromega/micromega.cmo \
+plugins/micromega/micromega.cmx \
+$(patsubst %.ml,%.ml.d,$(wildcard plugins/micromega/*.ml)) \
+$(patsubst %.mli,%.mli.d,$(wildcard plugins/micromega/*.mli)) \
+plugins/micromega/micromega_plugin.mlpack.d \
+   : plugins/micromega/MExtraction.vo
+
```

Comment:
I am wondering whether there is a more elegant way how to achieve the same result.